openff/interchange/components/interchange.py

```diff
@@ -196,15 +202,16 @@ def from_smirnoff(
             >>> from openff.toolkit.typing.engines.smirnoff import ForceField
             >>> mol = Molecule.from_smiles("CC")
             >>> mol.generate_conformers(n_conformers=1)
-            >>> top = Topology.from_molecules([mol])
```

Comment:
This is no different than your proposal but seeing it in code helped me appreciate how this unlocks some uses cases that completely skip the need for users to understand what a `Topology` is - but of course adds no friction to users that do want to make an manipulate them.